config_docker_certs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+sudo service docker stop
```

Comment:
Instead of repeating sudo in all the commands, the following check could be added?

if [ "$EUID" != "0" ]; then
    echo "This script requires root access. Run this with sudo."
    exit
fi